prac_06/guitar.py

```diff
@@ -0,0 +1,22 @@
+CURRENT_YEAR = 2017
```

Comment:
Why 2017?

Reply:
Did you just copy the solutions, or did you write this? I can't see why else you'd put 2017 here. Please let me know @AlexCampbellJCU .